pkg/scheduler/framework/plugins/tainttoleration/taint_toleration_test.go

```diff
@@ -260,29 +260,30 @@ func TestTaintTolerationScore(t *testing.T) {
 }
 
 func TestTaintTolerationFilter(t *testing.T) {
-	unschedulable := framework.NewStatus(framework.UnschedulableAndUnresolvable, ErrReasonNotMatch)
 	tests := []struct {
 		name       string
 		pod        *v1.Pod
 		node       *v1.Node
 		wantStatus *framework.Status
 	}{
 		{
-			name:       "A pod having no tolerations can't be scheduled onto a node with nonempty taints",
-			pod:        podWithTolerations("pod1", []v1.Toleration{}),
-			node:       nodeWithTaints("nodeA", []v1.Taint{{Key: "dedicated", Value: "user1", Effect: "NoSchedule"}}),
-			wantStatus: unschedulable,
+			name: "A pod having no tolerations can't be scheduled onto a node with nonempty taints",
```

Comment:
Have you gofmt this file?

Reply:
yes, pull-kubernetes-verify test was failing without it